src/pages/home/base/assurance/api-reference.md

```diff
@@ -34,9 +34,11 @@ Flutter
 
 ## registerExtension
 
-<InlineAlert variant="warning" slots="text"/>
+<InlineAlert variant="warning" slots="header, text1"/>
 
-Deprecated as of 2.0.0. Please use the [MobileCore.registerExtensions](../mobile-core/api-reference.md#registerextensions) API instead.
+This API has been deprecated starting in v2.0.0 and removed in v3.0.0 of the Android extension.
```

Comment:
nit: `Android mobile extension` so it does not get confused with an extension of Android - same for all